courses/admin.py

```diff
@@ -73,8 +73,14 @@ class ElectiveCourseAdmin(admin.ModelAdmin):
     list_display = ('course', 'electives_set',)
 
 
+class TopicAdmin(admin.ModelAdmin):
+    """ModelAdmin for Programs"""
+    list_display = ('name',)
```

Comment:
Can we add this field to `list_filter` for `ProgramAdmin`?

Reply:
Done